lib/league_stat.rb

```diff
@@ -32,8 +32,10 @@ def game_stats_away(game)
     stats_by_team[game.away_team_id][:away_games] += 1
     if game.away_goals > game.home_goals
       stats_by_team[game.away_team_id][:away_wins] += 1
+      stats_by_team[game.away_team_id][:total_wins] += 1
```

Comment:
i added total wins to keep the calculations out of other methods, as it would make the methods bulky